src/mgmt/srv.pb-c.c

```diff
@@ -0,0 +1,1434 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
+/* Generated from: srv.proto */
+
+/* Do not generate deprecated warnings for self */
+#ifndef PROTOBUF_C__NO_DEPRECATED
+#define PROTOBUF_C__NO_DEPRECATED
+#endif
+
+#include "srv.pb-c.h"
+void   proto__empty_params__init
+                     (Proto__EmptyParams         *message)
+{
+  static const Proto__EmptyParams init_value = PROTO__EMPTY_PARAMS__INIT;
+  *message = init_value;
+}
+size_t proto__empty_params__get_packed_size
```

Comment:
Still wondering why all of your functions are prefixed with proto when the security ones aren't and we both use the proto package. I think its related to how you urn the protoc command to generate the source files. I think if you specify proto/mgmt.proto it will prefix them with proto where if you were in proto and just specified mgmt.proto it wouldn't have that header.

Reply:
@tanabarr, any words on this question, please?

Reply:
yes likely what David said above, is this something I can fix next time I have to generate the files?

Reply:
```
╭─tanabarr@c7imldev ~/projects/daos/src/proto/mgmt  ‹tanabarr/mgmt-split-proto*›
╰─➤  ~/protobuf/install/bin/protoc -I . --c_out=/home/tanabarr/projects/daos/src/mgmt srv.proto --plugin=/home/tanabarr/protobuf/install/bin/protoc-gen-c
```
still gives me the same result